### PR TITLE
fix(pattern): fixed image overlapping text on mobile

### PR DESCRIPTION
**Pull request by RaphaelAmadeu**
### Related Ticket(s)

#1943 

### Description

I've tried to fix an issue where the image from the lightbox would overlap the text on ios devices

### CODE REVIEW

**Comment by annawen1** [comment]
Screen Shot 2020-04-03 at 5 20 09 PM

@wonilsuhibm @kennylam is this something we are expecting? That the image will shrink for images with tall heights?

File: packages/react/src/components/LightboxMediaViewer/LightboxMediaViewer.js [hunk 1]
```diff
@@ -44,7 +44,10 @@ const LightboxMediaViewer = ({ media, ...modalProps }) => {
                 {media.type === 'video' ? (
                   <VideoPlayer videoId={media.src} />
                 ) : (
-                  <Image defaultSrc={media.src} alt={media.alt} />
+                  <div
+                    className={`${prefix}--lightbox-media-viewer__media-image-wrapper`}>
```
Comment (jeffchew) [inline packages/react/src/components/LightboxMediaViewer/LightboxMediaViewer.js:48]: @RaphaelAmadeu I was looking around and couldn't find this class, is this being used?

Comment (RaphaelAmadeu) [reply packages/react/src/components/LightboxMediaViewer/LightboxMediaViewer.js]: @jeffchew ,
This was part of a fix try,
I've removed in on the latest commit,
Thank you!

**Comment by jeffchew** [comment]
@RaphaelAmadeu there are merge conflicts after I pulled in another PR. Can you resolve?

**Pull request merged**